src/com/google/javascript/jscomp/BranchCoverageInstrumentationCallback.java

```diff
@@ -49,6 +49,11 @@ public BranchCoverageInstrumentationCallback(
   public void visit(NodeTraversal traversal, Node node, Node parent) {
     String fileName = traversal.getSourceName();
 
+    // If origin of node is not from sourceFile, do not instrument
```

Comment:
Add more context here - why don't we instrument? When does this happen?

Reply:
Updated.